systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaConnectUtils.java

```diff
@@ -58,7 +58,8 @@ public static void waitForMessagesInKafkaConnectFileSink(String kafkaConnectPodN
     }
 
     public static void waitForMessagesInKafkaConnectFileSink(String kafkaConnectPodName, String sinkFileName) {
```

Comment:
Are you sure it's not breaking changes for Connect tests?

Reply:
Good point, I change all dependencies on that matching string. Now it will do not break anything.